consensus/polybft/polybft.go

```diff
@@ -220,6 +216,11 @@ func (p *Polybft) Start() error {
 		return fmt.Errorf("consensus runtime start - restart epoch failed: %w", err)
 	}
 
+	// subscribe to consensus engine messages
+	if err := p.subscribeToIbftTopic(); err != nil {
+		return fmt.Errorf("topic subscription failed: %w", err)
+	}
```

Comment:
This change is the most important here:
prevent IBFT messages receiving before we are able to process it (consensus runtime, epoch metadata etc. is fully initialized)

Reply:
Update:
This is most likely not necessary when #1047 gets merged, because it calls `restartEpoch` when creating a `consensusRuntime` instance (https://github.com/0xPolygon/polygon-edge/pull/1047/files#diff-e5b2e93010ff84c3777aef136e0834d710dd22861ca62ffbab9d5baa12a52b3cR145)